app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills += skill.split(", ")
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.learn_skill("JavaScript, HTML, CSS")
```

Comment:
you can make it easier without calling a method "learn_skill", but use for example ".+= [...]"

Reply:
So what for we create a method learn_skill in this task, if we never use it?